.github/workflows/tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Open Source Programs Run Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Run tests
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: osp
+          POSTGRES_PASSWORD: osp
+          POSTGRES_DB: osp
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    
+    - name: Installing dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    
+    - name: Creating Zulip API key
+      run: echo -e "${{ secrets.ZULIP_API_KEY }}" >> download
+
+    - name: Migrating Test Database
+      run: python manage.py migrate
+
+    - name: Run tests
+      run: python manage.py test --keepdb
```

Comment:
Why you use `keepdb` here?

Reply:
Very good questions 🕵🏾‍♀️

Reply:
I use --keepdb to preserve the test database in my comuter. But now I think it's of no use here :sweat_smile: 
I will remove it.